dispatch/static/manager/src/js/reducers/AuthReducer.js

```diff
@@ -38,4 +39,12 @@ reducer.handle(fulfilled(types.AUTH.DELETE_TOKEN), () => {
   return initialState
 })
 
+reducer.handle(fulfilled(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: true })
+})
+
+reducer.handle(rejected(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: false })
```

Comment:
Would it be possible to simply *remove* the token altogether, instead of using an extra `validToken` variable to keep track of its validity? I think if the server says the token is invalid, then the frontend should immediately remove it from the browser. Thoughts?